lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -76,8 +76,13 @@ def clear_output
         def num_lines
           # @options will be an array of questions but each option can be multi-line
           # so to get the # of lines, you need to join then split
+
+          # empty_option_count is needed since empty option titles are omitted when
+          # reject(&:empty?) called
+
+          empty_option_count =  @options.count { |option| option.empty? }
```

Comment:
`@options.count(&:empty)`, when you just call a method on the object that gets given to the block, you can omit the entire thing and just call it with `&:method` #rubyThings